.github/workflows/ci.yml

```diff
@@ -189,7 +189,7 @@ jobs:
           script: |
             if (${{ env.BRANCH_MEASUREMENT }} !== ${{ env.MAIN_MEASUREMENT }}) {
               try {
-                github.rest.issues.createComment({
+                await github.rest.issues.createComment({
```

Comment:
Thanks for fixing this! The warning is showing up as expected now, so this change is exactly what was missing.